src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.TrailingHeaders.cs

```diff
@@ -15,6 +16,13 @@ namespace System.Net.Http.Functional.Tests
 {
     public abstract class HttpClientHandlerTest_TrailingHeaders_Test : HttpClientHandlerTestBase
     {
+        private static byte[] DataBytes = Encoding.ASCII.GetBytes("data");
+        private static IList<HttpHeaderData> TrailingHeaders = new HttpHeaderData[] {
+            new HttpHeaderData("MyCoolTrailerHeader", "amazingtrailer"), new HttpHeaderData("Hello", "World") };
```

Comment:
I'm amused that my terrible choice in random name/values is being propagated into new tests :)